hamming/hamming.py

```diff
@@ -0,0 +1,15 @@
+"""Calculate the Hamming difference between two DNA strands."""
+
+
+def hamming(strand1, strand2):
+
+    count = 0
+    if len(strand1) == len(strand2):
```

Comment:
So this is a good check here. If I were designing this, I'd probably raise an exception right off the bat if the strands were not of equal length. This is a guard clause and would explain _why_ the inputs are incorrect.

Reply:
Also think about the fact that the function is returning an `int` in one case and a Boolean in the other case. I'd recommend making your functions return similar types for any case. For example, a function should only be able to return `True` or `False`, or perhaps just a string or just a number. Then you can use exceptions to handle exceptional cases. But make sure they are exceptional, which takes judgement.

Reply:
@paulghaddad Great info!

I added in the exception: 

`if len(strand1) != len(strand2):
        raise Exception("The lists are different lengths")`


So should I delete the test (which i updated to return an int, instead of a bool):

```
def test_unequal_dna_strand_lengths():
    assert hamming("GGAG", "CATCG") == 0
```

Otherwise, this test fails.